info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T14:32:19.855Z"}
info: Connected to the database. {"timestamp":"2022-08-03T14:32:22.546Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T14:37:16.347Z"}
info: Connected to the database. {"timestamp":"2022-08-03T14:37:18.983Z"}
error: 
    status - 500 
    message - SyntaxError: Unexpected string in JSON at position 32
    at JSON.parse (<anonymous>)
    at parse (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\body-parser\lib\types\json.js:89:19)
    at C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\body-parser\lib\read.js:128:18
    at AsyncResource.runInAsyncScope (node:async_hooks:202:9)
    at invokeCallback (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\raw-body\index.js:231:16)
    at done (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\raw-body\index.js:220:7)
    at IncomingMessage.onEnd (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\raw-body\index.js:280:7)
    at IncomingMessage.emit (node:events:539:35)
    at IncomingMessage.emit (node:domain:475:12)
    at endReadableNT (node:internal/streams/readable:1345:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) 
    url - /api/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-03T14:37:30.590Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T14:39:54.090Z"}
info: Connected to the database. {"timestamp":"2022-08-03T14:39:56.771Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T14:40:12.864Z"}
info: Connected to the database. {"timestamp":"2022-08-03T14:40:15.404Z"}
error: 
    status - 500 
    message - TypeError: console.error is not a constructor
    at _callee$ (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\src\services\/user.service.js:12:11)
    at tryCatch (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-03T14:56:02.045Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T14:56:24.743Z"}
info: Connected to the database. {"timestamp":"2022-08-03T14:56:24.779Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T14:56:26.709Z"}
info: Connected to the database. {"timestamp":"2022-08-03T14:56:26.734Z"}
error: 
    status - 500 
    message - ReferenceError: error is not defined
    at _callee$ (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\src\services\/user.service.js:4:34)
    at tryCatch (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-03T14:56:33.740Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T15:01:05.391Z"}
info: Connected to the database. {"timestamp":"2022-08-03T15:01:05.421Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T15:02:24.009Z"}
info: Connected to the database. {"timestamp":"2022-08-03T15:02:24.034Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T15:02:50.250Z"}
info: Connected to the database. {"timestamp":"2022-08-03T15:02:50.277Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T15:03:37.338Z"}
info: Connected to the database. {"timestamp":"2022-08-03T15:03:37.369Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T15:06:28.591Z"}
info: Connected to the database. {"timestamp":"2022-08-03T15:06:28.631Z"}
error: 
    status - 500 
    message - ValidationError: User validation failed: lastName: Path `lastName` is required., firstName: Path `firstName` is required.
    at model.Document.invalidate (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\mongoose\lib\document.js:2782:32)
    at C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\mongoose\lib\document.js:2574:17
    at C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\mongoose\lib\schematype.js:1273:9
    at processTicksAndRejections (node:internal/process/task_queues:78:11) 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-03T15:07:08.184Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T15:07:41.396Z"}
info: Connected to the database. {"timestamp":"2022-08-03T15:07:41.426Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T15:07:44.352Z"}
info: Connected to the database. {"timestamp":"2022-08-03T15:07:44.378Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-03T15:07:51.748Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T15:12:00.957Z"}
info: Connected to the database. {"timestamp":"2022-08-03T15:12:00.987Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T15:12:03.376Z"}
info: Connected to the database. {"timestamp":"2022-08-03T15:12:03.410Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-03T15:12:07.658Z"}
info: Connected to the database. {"timestamp":"2022-08-03T15:12:07.690Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:24:44.435Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:24:44.467Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:24:55.924Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:24:55.958Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:24:59.300Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:24:59.322Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:25:20.825Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:25:20.848Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:25:47.498Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:25:47.524Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:26:16.434Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:26:16.460Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:26:22.168Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:26:22.193Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:26:38.944Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:26:38.963Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:26:45.555Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:26:45.579Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:26:56.232Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:26:56.260Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:27:15.372Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:27:15.395Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:30:55.455Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:30:55.478Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:31:34.782Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:31:34.808Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:31:39.516Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:31:39.545Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:31:54.694Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:31:54.718Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:31:59.203Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:31:59.244Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:32:15.647Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:32:15.675Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:32:36.081Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:32:36.110Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:32:44.221Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:32:44.247Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:32:47.975Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:32:47.997Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:32:52.244Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:32:52.268Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:33:03.609Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:33:03.634Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:33:20.426Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:33:20.447Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:33:24.047Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:33:24.076Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:33:26.994Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:33:27.016Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:33:31.705Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:33:31.733Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:33:45.939Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:33:45.966Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:33:50.381Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:33:50.405Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:34:15.087Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:34:15.112Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:34:21.207Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:34:21.233Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:34:30.241Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:34:30.265Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:34:50.900Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:34:50.927Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-04T04:35:38.842Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/users 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-04T04:37:24.534Z"}
error: uncaughtException: error is not defined
ReferenceError: error is not defined
    at _callee3$ (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\src\controllers\/user.controller.js:59:19)
    at tryCatch (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"date":"Thu Aug 04 2022 10:07:40 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":426508},"process":{"argv":["node","C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\src\\index.js"],"cwd":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18301162,"external":19972479,"heapTotal":49102848,"heapUsed":44315264,"rss":81362944},"pid":16244,"uid":null,"version":"v16.15.1"},"stack":"ReferenceError: error is not defined\n    at _callee3$ (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\src\\controllers\\/user.controller.js:59:19)\n    at tryCatch (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:86:17)\n    at Generator._invoke (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:66:24)\n    at Generator.next (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:117:21)\n    at asyncGeneratorStep (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:25:9)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","timestamp":"2022-08-04T04:37:40.626Z","trace":[{"column":19,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\src\\controllers\\/user.controller.js","function":"_callee3$","line":59,"method":null,"native":false},{"column":17,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"tryCatch","line":86,"method":null,"native":false},{"column":24,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"Generator._invoke","line":66,"method":"_invoke","native":false},{"column":21,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"Generator.next","line":117,"method":"next","native":false},{"column":24,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js","function":"asyncGeneratorStep","line":3,"method":null,"native":false},{"column":9,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js","function":"_next","line":25,"method":null,"native":false},{"column":5,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":96,"method":null,"native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:38:44.079Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:38:44.101Z"}
error: uncaughtException: error is not defined
ReferenceError: error is not defined
    at _callee3$ (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\src\controllers\/user.controller.js:59:19)
    at tryCatch (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.throw (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _throw (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:29:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"date":"Thu Aug 04 2022 10:08:47 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":426576},"process":{"argv":["node","C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\src\\index.js"],"cwd":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18901039,"external":22271527,"heapTotal":79249408,"heapUsed":50480392,"rss":113565696},"pid":28996,"uid":null,"version":"v16.15.1"},"stack":"ReferenceError: error is not defined\n    at _callee3$ (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\src\\controllers\\/user.controller.js:59:19)\n    at tryCatch (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:86:17)\n    at Generator._invoke (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:66:24)\n    at Generator.throw (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:117:21)\n    at asyncGeneratorStep (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _throw (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:29:9)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","timestamp":"2022-08-04T04:38:47.850Z","trace":[{"column":19,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\src\\controllers\\/user.controller.js","function":"_callee3$","line":59,"method":null,"native":false},{"column":17,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"tryCatch","line":86,"method":null,"native":false},{"column":24,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"Generator._invoke","line":66,"method":"_invoke","native":false},{"column":21,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"Generator.throw","line":117,"method":"throw","native":false},{"column":24,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js","function":"asyncGeneratorStep","line":3,"method":null,"native":false},{"column":9,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js","function":"_throw","line":29,"method":null,"native":false},{"column":5,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":96,"method":null,"native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:41:44.110Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:41:44.152Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:41:59.020Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:41:59.047Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:42:23.751Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:42:23.776Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:42:32.902Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:42:32.924Z"}
error: uncaughtException: error is not defined
ReferenceError: error is not defined
    at _callee2$ (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\src\controllers\/user.controller.js:39:19)
    at tryCatch (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"date":"Thu Aug 04 2022 10:13:25 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":426853},"process":{"argv":["node","C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\src\\index.js"],"cwd":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18301436,"external":19965808,"heapTotal":48054272,"heapUsed":44114760,"rss":80977920},"pid":25620,"uid":null,"version":"v16.15.1"},"stack":"ReferenceError: error is not defined\n    at _callee2$ (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\src\\controllers\\/user.controller.js:39:19)\n    at tryCatch (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:86:17)\n    at Generator._invoke (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:66:24)\n    at Generator.next (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:117:21)\n    at asyncGeneratorStep (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:25:9)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","timestamp":"2022-08-04T04:43:25.361Z","trace":[{"column":19,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\src\\controllers\\/user.controller.js","function":"_callee2$","line":39,"method":null,"native":false},{"column":17,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"tryCatch","line":86,"method":null,"native":false},{"column":24,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"Generator._invoke","line":66,"method":"_invoke","native":false},{"column":21,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"Generator.next","line":117,"method":"next","native":false},{"column":24,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js","function":"asyncGeneratorStep","line":3,"method":null,"native":false},{"column":9,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js","function":"_next","line":25,"method":null,"native":false},{"column":5,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":96,"method":null,"native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:46:33.604Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:46:33.630Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:46:52.591Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:46:52.610Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:50:54.828Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:50:54.853Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:51:10.251Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:51:10.279Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:51:23.507Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:51:23.531Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:51:41.740Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:51:41.764Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:51:48.845Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:51:48.868Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:51:55.013Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:51:55.033Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:51:59.147Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:51:59.175Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:52:21.012Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:52:21.042Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:54:38.559Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:54:38.587Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-04T04:55:21.025Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-04T04:55:38.929Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:58:34.922Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:58:34.950Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:58:38.580Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:58:38.604Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:58:46.580Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:58:46.608Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:58:51.379Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:58:51.401Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-04T04:59:26.961Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-08-04T04:59:36.496Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T04:59:58.505Z"}
info: Connected to the database. {"timestamp":"2022-08-04T04:59:58.540Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T05:07:08.334Z"}
info: Connected to the database. {"timestamp":"2022-08-04T05:07:08.361Z"}
error: uncaughtException: error is not defined
ReferenceError: error is not defined
    at _callee2$ (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\src\controllers\/user.controller.js:39:19)
    at tryCatch (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.throw (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _throw (C:\Urmila data\data\WorkSpace\FundooProject\FundooNotes\fundoo_notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:29:9)
    at processTicksAndRejections (node:internal/process/task_queues:96:5) {"date":"Thu Aug 04 2022 10:43:24 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":428652},"process":{"argv":["node","C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\src\\index.js"],"cwd":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18307751,"external":19987133,"heapTotal":48840704,"heapUsed":44684528,"rss":81936384},"pid":32436,"uid":null,"version":"v16.15.1"},"stack":"ReferenceError: error is not defined\n    at _callee2$ (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\src\\controllers\\/user.controller.js:39:19)\n    at tryCatch (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:86:17)\n    at Generator._invoke (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:66:24)\n    at Generator.throw (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:117:21)\n    at asyncGeneratorStep (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _throw (C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:29:9)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)","timestamp":"2022-08-04T05:13:24.053Z","trace":[{"column":19,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\src\\controllers\\/user.controller.js","function":"_callee2$","line":39,"method":null,"native":false},{"column":17,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"tryCatch","line":86,"method":null,"native":false},{"column":24,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"Generator._invoke","line":66,"method":"_invoke","native":false},{"column":21,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js","function":"Generator.throw","line":117,"method":"throw","native":false},{"column":24,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js","function":"asyncGeneratorStep","line":3,"method":null,"native":false},{"column":9,"file":"C:\\Urmila data\\data\\WorkSpace\\FundooProject\\FundooNotes\\fundoo_notes\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js","function":"_throw","line":29,"method":null,"native":false},{"column":5,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":96,"method":null,"native":false}]}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T05:15:38.393Z"}
info: Connected to the database. {"timestamp":"2022-08-04T05:15:38.415Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T05:15:49.344Z"}
info: Connected to the database. {"timestamp":"2022-08-04T05:15:49.371Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2022-08-04T05:16:02.698Z"}
info: Connected to the database. {"timestamp":"2022-08-04T05:16:02.726Z"}
